src/smart-components/Conversation/components/ConversationScroll.jsx

```diff
@@ -153,6 +153,8 @@ export default class ConversationScroll extends Component {
                             emojiContainer,
                             chainTop,
                             chainBottom,
+                            hasSeparator,
+                            menuDisabled: disabled,
```

Comment:
I think adding hasSeparator makes sense, but can't be sure about menuDisabled. What do you think about @sravan-s